lib/commits.js

```diff
@@ -1,78 +1,99 @@
 const log = require('./log')
+const paginate = require('./pagination')
+const _ = require('lodash')
 
-module.exports.findCommits = async ({ app, context, branch, lastRelease }) => {
+const findCommitsWithAssociatedPullRequestsQuery = /* GraphQL */ `
+  query findCommitsWithAssociatedPullRequests(
+    $name: String!
+    $owner: String!
+    $branch: String!
+    $since: GitTimestamp
+    $after: String
+  ) {
+    repository(name: $name, owner: $owner) {
+      ref(qualifiedName: $branch) {
+        target {
+          ... on Commit {
+            history(first: 100, since: $since, after: $after) {
+              totalCount
+              pageInfo {
+                hasNextPage
+                endCursor
+              }
+              nodes {
+                id
+                message
+                author {
+                  name
+                  user {
+                    login
+                  }
+                }
+                associatedPullRequests(first: 5) {
+                  nodes {
+                    title
+                    number
+                    author {
+                      login
+                    }
+                    labels(first: 10) {
+                      nodes {
+                        name
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`
+
+module.exports.findCommitsWithAssociatedPullRequests = async ({
+  app,
+  context,
+  branch,
+  lastRelease
+}) => {
+  const { owner, repo } = context.repo()
+  const variables = { name: repo, owner, branch }
+  const dataPath = ['repository', 'ref', 'target', 'history']
+
+  let data
   if (lastRelease) {
     log({
       app,
       context,
-      message: `Comparing commits ${lastRelease.tag_name}..${branch}`
+      message: `Fetching all commits for branch ${branch} since ${
+        lastRelease.published_at
```

Comment:
looks funky

Reply:
Yeah looks a bit weird, it happens because the max line length is reached. There's not much I can do about it except increasing the line length 😬. I personally don't really mind though.